turn.yml

```diff
@@ -0,0 +1,27 @@
+version: '3'
+
+services:
+    # coturn TURN server project
+    turn:
+        image: jitsi/turn
```

Comment:
let's call it coturn, same way we name each image after the project it uses, for the most part.

Reply:
IMO, actually, I would prefer to stay it "turn" because it fully describes the functionality of the service. Coturn is a "coturn TURN server project", so I don't see any reasons to rename it. WDYS

Reply:
Ok.